app/src/main/java/com/example/trempelapp/dataLayer/inMemory/UserInfoRepositoryImpl.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.trempelapp.dataLayer.inMemory
+
+import com.example.trempelapp.dataLayer.inMemory.UserInfoRepository
+import javax.inject.Inject
+
+class UserInfoRepositoryImpl @Inject constructor(
+/*private val preferences: SharedPreferences*/
```

Comment:
Pass SharedPreferencesManager, And API service which was generated by retrofit

Reply:
done